tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -271,6 +273,8 @@ def update_snapshot(self, data: bytes) -> None:
 
         Args:
             data: unverified new snapshot metadata as bytes
+            trusted: whether to consider the given data as trusted. Default is
+                False.
```

Comment:
please try rephrasing this: current text tells the reader nothing on top of the variable name. Try answering the questions "what does trusted mean?" and/or "when should I set it to True?" -- might not be easy given how complex the use case is